morpheus/stages/input/http_client_source_stage.py

```diff
@@ -94,10 +97,11 @@ def __init__(self,
                  error_sleep_time: float = 0.1,
                  respect_retry_after_header: bool = True,
                  request_timeout_secs: int = 30,
-                 accept_status_codes: typing.List[HTTPStatus] = (HTTPStatus.OK, ),
+                 accept_status_codes: list[HTTPStatus] = (HTTPStatus.OK, ),
```

Comment:
The type is a list so the default value should be a list

```suggestion
                 accept_status_codes: list[HTTPStatus] = [HTTPStatus.OK],
```

Reply:
The reason why I initialized it with a tuple is that it's an immutable type, I was explicitly attempting to avoid the problems in Python when the default value is a mutable type.

Reply:
Convert the type hint to be iterable